adoc/MAIN.release-notes.adoc

```diff
@@ -23,6 +23,34 @@ To make such entries easier to identify, they contain a note to that effect.
 
 {vmware} {tf} config now supports setting a datastore cluster as the storage backend. Please refer to the  link:https://documentation.suse.com/suse-caasp/4.2/html/caasp-deployment/_deployment_instructions.html#_deploying_vms_from_the_template[Deployment Instructions] for more information.
 
+=== Addon customization persistence
+
+This release adds `kustomize` to persist addon configuration changes across updates and reboots.
+
+=== Required Actions
+
+==== Generate the `kustomize` style addon configurations
+
+You must convert your addon manifests to the new `kustomize` aware file structure and formats.
+To do so please run the following command from your management workstation.
+
+Replace `MASTER-NODE-IP` with an IP address/FQDN of one of your master nodes.
+Replace `CLUSTER-DEFINITION-PATH` with the path to your existing cluster defintion files that were generated during the intial bootstrap/deployment.
+
+----
+skuba cluster init --control-plane MASTER-NODE-IP /tmp/new-cluster-init
+cp /tmp/new-cluster-init/addons CLUSTER-DEFINITION-PATH
```

Comment:
I would change it to 
`mv CLUSTER-DEFINITION-PATH/addons CLUSTER-DEFINITION-PATH/addons-old`
`cp -r /tmp/mjura-cluster/addons CLUSTER-DEFINITION-PATH/`